vtysh/vtysh_ovsdb_router_context.c

```diff
@@ -195,13 +195,23 @@ static void vtysh_router_context_bgp_print_nbr_cfg(vtysh_ovsdb_cbmsg_ptr p_msg,
 
     if (nbr_table->update_source) {
         if (!pgroup_ptr ||
-            strcmp(nbr_table->update_source, pgroup_ptr->update_source)  != 0){
+            strcmp(nbr_table->update_source, pgroup_ptr->update_source) != 0){
             vtysh_ovsdb_cli_print(p_msg,"%4s %s %s %s %s", "", "neighbor",
                                  neighbor,
                                  "update-source",
                                  (nbr_table->update_source));
         }
     }
+
+    if (nbr_table->local_as) {
```

Comment:
Yes, I propose to maintain this style everywhere. In my opinion the number of records is more adequate criterion, than the value of the pointer.